services/merak-topo/handler/topo_deploy.go

```diff
@@ -38,12 +38,15 @@ import (
 )
 
 var (
-	ACA_IMAGE = "phudtran/merak-agent:dev"
+	// ACA_IMAGE = "phudtran/merak-agent:dev"
+	ACA_IMAGE = "meraksim/merak-agent:dev"
+	// ACA_IMAGE = "meraksim/merak-agent:testv2"
 	OVS_IMAGE = "yanmo96/ovs_only:latest"
-	GW_IMAGE  = "yanmo96/aca_build_standard:v2"
-	RYU_IP    = "ryu.default.svc.cluster.local"
-	RYU_PORT  = "6653"
-	Ctx       = context.Background()
+	// GW_IMAGE  = "yanmo96/aca_build_standard:v2"
+	RYU_IP = "ryu.default.svc.cluster.local"
+	// RYU_IP   = "10.213.43.77"
+	RYU_PORT = "6653"
```

Comment:
@zzengcs Try to use environment variable or config file to get these parameters in the next PR.

Reply:
Delete the comment parts in current PR